aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -393,7 +393,6 @@ Tensor& logical_not_(Tensor& self) {
 Tensor& logical_not_out(Tensor& result, const Tensor& self) {
   TensorIterator iter = TensorIteratorConfig()
     .check_all_same_dtype(false)
-    .set_check_mem_overlap(true)
```

Comment:
nit: signbit_out below also checks memory overlap, so we can delete that line. (line 413/414)

Reply:
Had a quick grep for `.set_check_mem_overlap(true)` outside of `TensorIterator`.